e2e/bitrise.yml

```diff
@@ -8,8 +8,23 @@ workflows:
     after_run:
     - _check_version
     steps:
+    - script:
+        title: Check if JDK version is available on stack
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            if [[ "$BITRISEIO_STACK_ID" == osx-xcode-15.* ]] || [[ "$BITRISEIO_STACK_ID" == osx-xcode-14.* ]] || [[ "$BITRISEIO_STACK_ID" == linux-docker-android-* ]]; then
```

Comment:
I think you already explained earlier somewhere that the edge stack is like on alias to the latest beta stack but I thought I will still ask that is it good that the edge stack is missing from this if?